Hw3.py

```diff
@@ -0,0 +1,42 @@
+# Home task_1
+test_word = str(input('Enter any word:'))
```

Comment:
It does not make sense in str(input()) , since in input() it is already a string by default.

Reply:
+